assets/textures/gui/bevel/CurrentDilution.svg.import

```diff
@@ -0,0 +1,37 @@
+[remap]
+
+importer="texture"
+type="CompressedTexture2D"
+uid="uid://byel4jeqodw5r"
+path="res://.godot/imported/CurrentDilution.svg-911f83b993c49654b128c2b2aab019c9.ctex"
+metadata={
+"vram_texture": false
+}
+
+[deps]
+
+source_file="res://assets/textures/gui/bevel/CurrentDilution.svg"
+dest_files=["res://.godot/imported/CurrentDilution.svg-911f83b993c49654b128c2b2aab019c9.ctex"]
+
+[params]
+
+compress/mode=0
+compress/high_quality=false
+compress/lossy_quality=0.7
+compress/hdr_compression=1
+compress/normal_map=0
+compress/channel_pack=0
+mipmaps/generate=true
+mipmaps/limit=-1
+roughness/mode=0
+roughness/src_normal=""
+process/fix_alpha_border=true
+process/premult_alpha=false
+process/normal_map_invert_y=false
+process/hdr_as_srgb=false
+process/hdr_clamp_exposure=false
+process/size_limit=0
```

Comment:
Are the SVGs naturally small or? I think the size limit ought to be set to 128 or 256 to ensure no unnecessary graphics memory use.